tests/desktop/consumer_pages/test_users_account.py

```diff
@@ -35,78 +33,61 @@ def test_user_can_sign_in_and_sign_out_from_home_page(self, mozwebqa, new_user):
         home_page.go_to_homepage()
         home_page.header.click_sign_in()
         home_page.login(new_user['email'], new_user['password'])
-        Assert.true(home_page.is_the_current_page)
-
-        # Verify that user is logged in
-        Assert.true(home_page.header.is_user_logged_in)
+        assert home_page.is_the_current_page
+        assert home_page.header.is_user_logged_in
 
         home_page.header.click_sign_out()
-        Assert.true(home_page.header.is_sign_in_visible)
+        assert home_page.header.is_sign_in_visible
 
     @pytest.mark.nondestructive
     def test_user_can_sign_in_and_sign_out_from_my_apps(self, mozwebqa, new_user):
         my_apps_page = My_Apps(mozwebqa)
         my_apps_page.go_to_my_apps_page()
-
         my_apps_page.click_sign_in()
         my_apps_page.login(new_user['email'], new_user['password'])
-
-        Assert.true(my_apps_page.header.is_user_logged_in)
-        Assert.false(my_apps_page.header.is_sign_in_visible)
+        assert my_apps_page.header.is_user_logged_in
+        assert not my_apps_page.header.is_sign_in_visible
 
         my_apps_page.header.click_sign_out()
-        Assert.true(my_apps_page.header.is_sign_in_visible)
+        assert my_apps_page.header.is_sign_in_visible
 
     @pytest.mark.nondestructive
     def test_user_can_sign_in_and_sign_out_from_settings_page(self, mozwebqa, new_user):
         settings_page = AccountSettings(mozwebqa)
         settings_page.go_to_settings_page()
-
         settings_page.click_sign_in()
         settings_page.login(new_user['email'], new_user['password'])
-
-        Assert.true(settings_page.header.is_user_logged_in)
-        Assert.false(settings_page.header.is_sign_in_visible)
+        assert settings_page.header.is_user_logged_in
+        assert not settings_page.header.is_sign_in_visible
 
         settings_page.click_sign_out()
-        Assert.true(settings_page.header.is_sign_in_visible)
+        assert settings_page.header.is_sign_in_visible
 
     def test_editing_user_profile(self, mozwebqa, new_user):
         home_page = Home(mozwebqa)
         home_page.go_to_homepage()
         home_page.header.click_sign_in()
         home_page.login(new_user['email'], new_user['password'])
-
         profile_page = home_page.header.click_edit_account_settings()
+        assert profile_page.email.split('@')[0] == profile_page.display_name
 
-        # Initial check
-        Assert.equal(profile_page.email.split('@')[0], profile_page.display_name)
-
-        # Data to submit
         name = 'Napoleon'
-
         profile_page.edit_display_name(name)
         profile_page.save_changes()
         profile_page.wait_for_notification('Your settings have been successfully saved')
-
-        # Refresh page and then inspect saved settings
         profile_page.refresh_page()
-        Assert.equal(profile_page.display_name, name)
+        assert name == profile_page.display_name, name
```

Comment:
I think there's a typo here. Did you mean to leave the extra `, name` on the end?